examples/pytorch/distributed_stl10/pytorch_distributed_stl10_example.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#SBATCH --job-name=pytorch_stl10     # job name
+#SBATCH --ntasks=2                   # number of MP tasks
+#SBATCH --ntasks-per-node=1          # number of MPI tasks per node
+#SBATCH --gres=gpu:1                 # number of GPUs per node
+#SBATCH --cpus-per-task=10           # number of cores per tasks
+#SBATCH --hint=nomultithread         # we get physical cores not logical
+#SBATCH --time=00:10:00              # maximum execution time (HH:MM:SS)
+#SBATCH --output=pytorch_stl10%j.out # output file name
+#SBATCH --error=pytorch_stl10%j.err  # error file name
+
+set -x
+cd $WORK/jean-zay-doc/examples/pytorch/distributed_stl10
```

Comment:
```suggestion
cd ${SLURM_SUBMIT_DIR}
```

Reply:
I agree with these changes. The only reason I did it this way was to stay consistent with the other torch example. We should push this changes to both example then.

Reply:
Yes, you're right. I'll modify the other example to keep consistency.